src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
The class name `Rectangle` is appropriate as it is a noun. Good job on naming here.